files/task_helper.rb

```diff
@@ -18,7 +18,7 @@ def to_h
     end
   end
 
-  def task(params = {})
+  def task(params = {}, remote = nil)
```

Comment:
Does this need to be it's own argument to the task helper, or can it just be a keyword argument that's passed in when needed?

Reply:
I'm happy to change it if you have a preference, could you give me an example of how you would like this to look?

Reply:
Yeah this won't work long term when we may have other optional data that is important to the task implementation. 

I think making `remote` an instance method is probably a better API